<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pareto_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить множество Парето</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_n_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_m_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>m =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t1_min">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t1min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t2min_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t2min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t3min_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t3min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_p_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>p =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_scalar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скалярная свертка критерия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a1_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>a1 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a2_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>a2 =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_target">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод прицельной точки:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_a1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_a2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_n">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_m">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_p">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t3min">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t3max">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t1min">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t2min">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t1max">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t2max">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_input">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Блок ввода параметров</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t1_min_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t1max =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t2max_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t2max =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t3max_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>t3max =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать критерии</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_scalar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_target">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_intervals_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество интервалов =</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="input_interval_count">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
